src/Searchbar.vue

```diff
@@ -19,14 +19,19 @@ export default {
         return undefined;
       }
       const matches = [];
-      const regex = new RegExp(this.value, 'i');
+      const regexes = this.value.split(' ').map(searchKeyword => new RegExp(searchKeyword, 'i'));
       this.data.forEach((entry) => {
         const { headings, src, title } = entry;
         const keywords = entry.keywords || '';
-        let hasMatchingHeading = false;
+
+        let searchTarget = [title].concat(keywords).join(' ');
+        if (regexes.every(regex => regex.test(searchTarget))) {
+          matches.push(entry);
```

Comment:
We should set `isMatchingPage = true`. As I mentioned:

> If `title` and `keywords` match, all headings will be matched.

Reply:
In that case we push all entries

Reply:
one entry for the full page, then entries for all headings